### ISSUE TITLE
Bad left-side bearings of mathematical italic glyphs

### ISSUE THREAD

**Issue by ivo-s** [libertinus-fonts/libertinus#252]
First, let me thank you for your work on the Libertinus typeface, it looks great!

Originally, I noticed some odd kerning when using Libertinus Math in LuaLaTeX / XeLaTeX. After some research, I think that the fault is with font bearings. Sometimes, they are very asymmetrical. The case where it really stands out are the brackets. This is a result of typesetting using the libertinus package in XeLaTeX:

![libmath](https://user-images.githubusercontent.com/7095181/50741523-78596280-11fe-11e9-9658-3ef2b322219d.png)

The bearings seem to originate from Libertinus Italic, but in math typesetting I think they should be more optimized for the use with upright glyphs. In Latin Modern Math, the bearings are more symmetrical (assuming italic correction).

![lsb_comparison](https://user-images.githubusercontent.com/7095181/50741677-96c05d80-1200-11e9-99e2-325b80e29880.png)

I did some quick fiddling with the bearings in FontForge to make them more symmetrical and tested it:

![libmath_2](https://user-images.githubusercontent.com/7095181/50741721-83fa5880-1201-11e9-8d7f-1273fded193d.png)

While I can't say that I am sure that all the characters look fine in this particular context, letters like C, G, W, T, b or p look much better to me.

I am submitting an issue instead of a pull request for several reasons. First, I need to check that I am not wrong :-) I am no typographer and I have no experience with fonts. This means that I am also not sure what the correct bearings should be. They should probably not be precisely symmetrical, especially if there is more "meat" on one side of a glyph...? Another thing is that adjusting the left-side bearing means that some other entries need to be recalculated accordingly. I only know about top accent position and kerning lookup table, but is that all?

So, provided that the bearings should be fixed, it will take a bit of work. All the italic glyphs in the Unicode block 0x1D400 - 0x1D7FF should be checked, because I found asymmetrical bearings in multiple families (serif, sans, bold...). Then, a Python script should recalculate all the relevant parameters. If you don't have the time, I can try to help, although I would need advice on the issues above. Thanks!

**Comment by ivo-s** [comment]
Thanks for the info, I will trust my æsthetic judgement then :-) Let me clarify the script thing: I meant that I would adjust the LSB by hand, and then I would need to fix top accents and kerning tables by a script so that everything would be the same as before. That is because setting LSB does not change these values, but moves the glyph to different coordinates, at least that is what I noticed.

I also noticed some ongoing pull requests, is it going to be fine if I base my changes on the current master sources?
